.cirrus.yml

```diff
@@ -65,6 +65,7 @@ validate_task:
 
 
 cross_task:
+    only_if: $CI != $CI
```

Comment:
Is this intended to mean `false`, or is it something more subtle?

Reply:
-sigh- no, I asked them about this a very long time ago.  Cirrus-CI cannot spell `false` :cry: 

If it's any clearer, we could do something like `only_if: 0 == 1`

Reply:
\*shrug\* this works fine. I’m probably going to forget and keep asking anyway (it’s quite possible that I have asked about this sometime in the past, at least once, already).

It might be worth a comment, in principle, but this is a backport from `main`, so being consistent with the condition already there is at least as important, and this discussion already serves as a record.